.github/workflows/publish-26x-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and push WildFly Docker images
+
+# This action requires 4 secrets:
+# secrets.REGISTRY          - the container registry (e.g. quay.io)
+# secrets.IMAGE             - the namespaced name of the image (e.g. wildfly/wildfly)
+# secrets.REGISTRY_USER     - the user name to authentication before pushing the image
+# secrets.REGISTRY_PASSWORD - the user password to authentication before pushing the image
+
+on:
+  push:
+    tags:
+      - "26*"
+
+jobs:
+  image:
+    strategy:
+      matrix:
+        jdk-version: [11, 17]
+    environment: quay.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4.1.1
+        with:
+          images: ${{ secrets.REGISTRY }}/${{ secrets.IMAGE }}
+          flavor: |
+            latest=false
+          tags: |
+            type=ref,event=tag,suffix=-jdk${{ matrix.jdk-version }}
```

Comment:
This looks fine to me, but a general comment:

I think it should be possible to just extract the tag name from any new tag pushed, for example: 

```
run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
        echo $RELEASE_VERSION
        echo ${{ env.RELEASE_VERSION }}
```

Or something similar, thereby not needing to have specific files for each tag etc. I'm not so sure how often we'd end up doing this, so maybe the effort isn't worth it?

Reply:
Thanks @luck3y.

I'll file an issue to improve this. I went back and forth a bit on trying to do something more clever but didn't really have the time. Plus like you said I wasn't sure if this wasn't going to be more of a one-off, or limited to 26.x.

Once there's an issue if someone decides to do it because they're in the mood, fine; otherwise if the need for this pops up again someone may decide to do it.

Reply:
See #171